src/components/lco/PageBanner.tsx

```diff
@@ -36,62 +44,93 @@ type PageBannerProps = Pick<ClimbType, 'ancestors'>
  */
 export const PageBanner: React.FC<PageBannerProps> = ({ ancestors }) => {
   const orgs = findLCOs(LCO_LIST, ancestors)
-  if (orgs.length === 0) return null
   return (
-    <div className='mt-8 bg-area-cue/20 p-6 rounded-box'>
-      <div className='mt-2 mb-1 text-xs text-base-300'>Local climbing organizations</div>
-      <div className='flex flex-col'>
-        {orgs.map(orgProfile => <IndividualBanner key={orgProfile.id} profile={orgProfile} />)}
+    <div className='grid pt-6 pb-4 lg:pb-16 lg:pt-16'>
+      <div className='col-span-full flex justify-start items-center pb-6'>
+        <h3 className='mr-4'>Local climbing organizations</h3>
+        <InformationCircleIcon className='h-6 w-6' />
+      </div>
+      <div>
+        {orgs.length === 0
+          ? (
+            <p className='italic text-base-content/60'>
+              No organizationa found for this area
+            </p>
+            )
+          : (
+              orgs.map((orgProfile) => (
+                <IndividualBanner key={orgProfile.id} profile={orgProfile} />
+              ))
+            )}
       </div>
     </div>
   )
 }
 
 const IndividualBanner: React.FC<ContentProps> = ({ profile }) => (
-  <HoverCard.Root openDelay={300}>
-    <HoverCard.Trigger>
-      <ContentTrigger profile={profile} />
-    </HoverCard.Trigger>
-    <HoverCard.Portal>
-      <HoverCard.Content sideOffset={10} side='top' className='z-40'>
-        <Card profile={profile} />
-      </HoverCard.Content>
-    </HoverCard.Portal>
-  </HoverCard.Root>
+  <>
+    <div className='sm:inline-block mr-6 mb-6'>
+      <div className='flex items-center bg-light hover:bg-on-hover pl-5 pr-7 rounded-2xl'>
```

Comment:
Highlighting (darkening) of the LCO item on hover is a bit confusing to me as I gives me the impression that I can click on the whole div, but only the url is clickable.

Reply:
Yeah I think AJ's intention is to let the user click on the whole div and show the LCO profile card.